curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c90830bbeb40ab15148a6.md

```diff
@@ -0,0 +1,169 @@
+---
+id: 643c90830bbeb40ab15148a6
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+`forEach` is an array method in JavaScript that allows you to iterate over each element of an array and perform some operation on each element. Here's an example:
+
+```js
+const arr = [1, 2, 3, 4, 5];
+arr.forEach((element) => {
+  console.log(element);
+});
+```
+
+In this step, you will use the `forEach` method on the `topics` array, Like this 
+
+```js
+topics.forEach();
+```
```

Comment:
```suggestion
Call the `.forEach()` method on your `topics` array. Do not pass a callback function yet.
```

- Campers have already been taught forEach
- Examples should really never show the actual code campers should write.

Reply:
fixed